tests/PHPUnit/Framework/Fixture.php

```diff
@@ -615,6 +615,8 @@ public static function getRootUrl()
             $piwikUrl .= $piwikUri;
         }
 
+        $piwikUrl = getenv('GITHUB') ? "http://localhost/" : $piwikUrl;
```

Comment:
Why don't you simply add the `[tests] http_host=http://localhost` to the config file that is used on github instead?

Reply:
I think we you run php -S on one port, and try to curl the same port it actually returns an error. So I made the test run on port 80, but the actual test is on port 3000.